site/docs/Tutorial-ExpressGraphQL.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Running an Express GraphQL Server
+layout: ../_core/DocsLayout
+category: Tutorial
+permalink: /graphql-js/running-an-express-graphql-server/
+next: /graphql-js/graphql-clients/
+---
+
+The simplest way to run an GraphQL API server is to use Express, a popular web application framework for Node.js. (link to https://expressjs.com/). You will need to install two additional dependencies:
+
+```bash
+npm install express express-graphql --save
+```
+
+Let's modify our “hello world” example so that it's an API server rather than a script that runs a single query. We can use the 'express' module to run a webserver, and instead of executing a query directly with the `graphql` function, we can use the `express-graphql` library to mount a GraphQL API server on the “/graphql” HTTP endpoint:
+
+```javascript
+var express = require('express');
+var graphqlHTTP = require('express-graphql');
+var { buildSchema } = require('graphql');
+
+// Construct a schema, using GraphQL schema language
+var schema = buildSchema(`
+  type Query {
+    hello: String
+  }
+`);
+
+// The root provides a resolver function for each API endpoint
+var root = {
+  hello: () => {
+    return 'Hello world!';
+  },
+};
+
+var app = express();
+app.use('/graphql', graphqlHTTP({
+  schema: schema,
+  rootValue: root,
+  graphiql: true,
+}));
+app.listen(4000);
+console.log('Running a GraphQL API server at localhost:4000/graphql');
+```
+
+You can run this GraphQL server with:
+
+```bash
+node server.js
+```
+
+Since we configured `graphqlHTTP` with `graphiql: true`, you can use the GraphiQL tool to manually issue GraphQL queries. If you navigate in a web browser to `http://localhost:4000/graphql`, you should see an interface that lets you enter queries. It should look like:
```

Comment:
Hmm, that seems like a reasonable practice but I don't really want to make all of the example code include the extra concept of "what is NODE_ENV" because I think many people won't understand that or use that convention.